lang_java/parsing/parser_java.mly

```diff
@@ -1229,6 +1229,8 @@ interface_member_declaration:
  | annotation_type_declaration { [Class $1] }
 
  | ";"  { [] }
+ (* sgrep-ext: allows ... inside interface body *)
+ | "..." { [DeclEllipsis $1] }
```

Comment:
Hi @IagoAbal , good to see you here!

Can I ask that you add a test case?

You can drop it in `tests/java/semgrep`, or consider adding it in `tests/java/dots_interface.sgrep` in the `semgrep` repo when you rev the `pfff` commit.

Reply:
Sure, I can add a test now to this repo, and when this is merged and semgrep's submodule is updated, then I can add a test to semgrep too.